extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -116,6 +117,19 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
     );
   }
 
+  /**
+   * Resolves relative URLs to their full path.
+   * @param {string} url
+   * @private
+   * @return {string}
+   */
+  resolveRelativeUrl_(url) {
+    if (/^\//.test(url) && isSecureUrlDeprecated(url)) {
```

Comment:
No need for `isSecureUrl` if we test that it starts with `^/`.
```suggestion
    if (/^\//.test(url)) {
```

Alternatively, I'm not sure if we need the path check at all. Resolving the relative URL using absolute URL will return the original URL every time.
```suggestion
```